SystemServices/SystemServices.cpp

```diff
@@ -354,6 +358,9 @@ namespace WPEFramework {
 
             // version 2 APIs
             registerMethod(_T("getTimeZones"), &SystemServices::getTimeZones, this, {2});
+            registerMethod("fwPendingReboot", &SystemServices::fwPendingReboot, this, {2});
+            registerMethod("fwDelayReboot", &SystemServices::fwDelayReboot, this, {2});
+            registerMethod("fwAutoReboot", &SystemServices::fwAutoReboot, this, {2});
         }
```

Comment:
Change method names to:
setFirmwareAutoReboot
fireFirmwarePendingReboot
setFirmwareRebootDelay

Reply:
Added.